lib/model/migrations/20230104-01-add-entity-indices.js

```diff
@@ -0,0 +1,34 @@
+// Copyright 2023 ODK Central Developers
```

Comment:
I think the file should be named to have a date that comes after the latest existing migration otherwise knex might get confused (mainly for us devs updating between different commits in master, not necessarily someone doing a migration that includes all the 2023.1 files).

Reply:
Tried it, knex doesn't actually complain, but it will generally try to apply the migrations in the filename order and since it's getting committed after `20230106-01-remove-revision-number.js` and `20230109-01-add-form-schema.js` it would be safer to have it come after.